docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to get unleashed levels working in order to edit them, this guide does not cover how to use GLvl. its recommended you also read the [Generations GLvl guide.](/guides/hedgehog-engine/blueblur/levels/glvl.md){ target="_blank"}
+
+
+    
+!!! info
+
```

Comment:
This info section has blank lines, which shouldn't happen. Remove the empty lines in the info section (which also means no blank line right after `!!! info`)